actionpack/lib/action_controller/metal/rate_limiting.rb

```diff
@@ -6,21 +6,26 @@ module ActionController # :nodoc:
   module RateLimiting
     extend ActiveSupport::Concern
 
+    RateLimit = Struct.new(:name, :count, :retry_after) # :nodoc:
```

Comment:
This should be documented, no? Otherwise people will not know there is a name, count and retry_after methods in the request `rate_limit`. We should also document the methods in the request and response